api/api.go

```diff
@@ -55,3 +57,38 @@ type Config struct {
 	Hosts    []string `yaml:"hosts"`
 	Keyspace string   `yaml:"keyspace"`
 }
+
+// ProfilingAPI wraps an ordinary API and also records profiling metrics to a given Profiler object.
+type ProfilingAPI struct {
```

Comment:
should this go to `inspect`? I think I like the idea of `square/metrics/api` being a package without any dependency.